bft/src/main.rs

```diff
@@ -7,6 +7,6 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
     let args = cli::Cli::parse();
     let prog = Program::from_file(&args.program)?;
     let machine: Machine<u8> = Machine::new(args.cells, args.extensible);
-    machine.print_program(&prog);
+    machine.validate(&prog)?;
```

Comment:
Aha, here's why you no longer output the code, I was expecting both behaviours, not just the validation.